README.md

```diff
@@ -2,37 +2,34 @@
 
 The dotnet/core repository is a good starting point for .NET Core. 
 
-## Get Started
-
-If you're new to .NET Core, you can get started in 10 minutes: 
+## Learn about .NET Core 
 
-- [Get started with .NET Core](https://www.microsoft.com/net/core)
-- [Download .NET Core](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md)
+* [Learn about .NET Core](https://docs.microsoft.com/dotnet/core)
+* [.NET Core Roadmap](https://github.com/dotnet/core/blob/master/roadmap.md)
+* [Learn how about the .NET platform](https://docs.microsoft.com/dotnet/standard/)
 
-## Learn about .NET Core 
+## Download .NET Core
 
-- [Learn about .NET Core](https://docs.microsoft.com/dotnet/core)
-- [.NET Core Roadmap](https://github.com/dotnet/core/blob/master/roadmap.md)
-- [.NET Core release notes](https://github.com/dotnet/core/blob/master/release-notes/README.md)
-- [Learn how about the .NET platform](https://docs.microsoft.com/dotnet/standard/)
+* [Released builds](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md)
+* [.NET Core release notes](https://github.com/dotnet/core/blob/master/release-notes/README.md)
+* .NET Core 2.0 daily builds: 
+  * [.NET Core SDK 2.0 preview3](https://github.com/dotnet/cli/blob/release/2.0.0/README.md#installers-and-binaries)
+  * [.NET Core Runtime 2.0 preview3](https://github.com/dotnet/core-setup/blob/release/2.0.0/README.md#officially-released-builds)
+* .NET Core 2.1 daily builds:
```

Comment:
It should match what is printed in the shields here: https://github.com/dotnet/cli#installers-and-binaries. It isn't useful if they differ. I think it's fine to say 2.1 since we'll be talking about it very soon anyway. Fine question, BTW.